util/encryption.go

```diff
@@ -35,6 +41,14 @@ func InitializeEncryption() {
 	key = string(decoded)
 	keyPresent = true
 }
+func setDefaultEncryptionKey() error {
+	_ = os.Setenv("ENCRYPTION_KEY", "YWFhYWFhYWFhYWFhYWFhYQ")
+	getKey := os.Getenv("ENCRYPTION_KEY")
+	if getKey != "YWFhYWFhYWFhYWFhYWFhYQ" {
+		return fmt.Errorf("Wrong encryption key: did not set correct default value")
+	}
+	return nil
+}
```

Comment:
In this case I'd suggest not ignoring the error, and returning early. If the `Setenv` function returns an error, you know for sure that the value you wanted wasn't set.